src/config.js

```diff
@@ -438,4 +438,18 @@ export default Object.freeze({
     VAULT_STATUS: "active",
     CHAIN_ID: 1,
   },
+  ghostysusd: {
+    TITLE: "Ghosty Dollar",
+    LOGO: "𓀀💵",
+    VAULT_ABI: yVaultV2,
+    VAULT_TYPE: "experimental",
+    VAULT_ADDR: "0xa5cA62D95D24A4a350983D5B8ac4EB8638887396",
+    WANT_ADDR: "0x57Ab1ec28D129707052df4dF418D58a2D46d5f51",
+    WANT_SYMBOL: "SUSD",
+    COINGECKO_SYMBOL: "susd",
```

Comment:
The Coingecko ID is nusd. For some reason, the symbol and ID are sometimes different. You can check here: https://api.coingecko.com/api/v3/coins/ethereum/contract/0x57Ab1ec28D129707052df4dF418D58a2D46d5f51

```javascript
COINGECKO_SYMBOL: "nusd"
```